integrations.yml

```diff
@@ -238,4 +238,10 @@
   description: "Give users the right tools to collaborate and provide valuable feedback for you to manage."
   website: "https://www.userback.io"
   icon: "userback.png"
-  category: 5
\ No newline at end of file
+  category: 5
+-
+  name: "Juto"
+  description: "Juto gives your company its own mobile app. Create your forms on the Juto website, define a workflow for each form and link with your Basecamp To-do lists."
+  website: "http://ju.to/basecamp3.html"
```

Comment:
Please use `https://` since your site supports it.

Reply:
Well spotted @javan, I've updated it to https.